cluster-autoscaler/utils/kubernetes/listers.go

```diff
@@ -360,3 +360,14 @@ func NewStatefulSetLister(kubeClient client.Interface, stopchannel <-chan struct
 	go reflector.Run(stopchannel)
 	return lister
 }
+
+// NewConfigMapListerForNamespace builds a configmap lister for the passed namespace (including all).
+func NewConfigMapListerForNamespace(kubeClient client.Interface, stopchannel <-chan struct{},
+	namespace string) v1lister.ConfigMapLister {
+	listWatcher := cache.NewListWatchFromClient(kubeClient.AppsV1().RESTClient(), "configmaps", namespace, fields.Everything())
```

Comment:
Configmap is in core/v1, not apps/v1. Does that work for you? Seems like it shouldn't.

Reply:
Yeah, my bad, I ran tests actually before removing the test code here. I was sure it was exactly the same as in Listers - it wasn't. Fixing, and I'll re-run tests once more. Thanks for catching!

Reply:
@MaciekPytel please check now